packages/netlify-cms-widget-markdown/src/MarkdownPreview.js

```diff
@@ -1,17 +1,10 @@
-import React, { useEffect } from 'react';
+import React from 'react';
```

Comment:
@erquhart I ran into 2 missing prop-types errors on this component after the rebase.
One was solved by passing in `resolveWidget`. For the other one I couldn't find a usage for `editorPreview` nor `getEditorPreview ` so I removed both.